src/App.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setQuery(event.currentTarget.value);
+  }
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const lowerCaseInput = query
+      .toLowerCase()
+      .split(' ')
+      .filter(Boolean)
+      .join(' ');
```

Comment:
Better use this:
`const formattedSearchQuery = searchQuery.toLocaleLowerCase().trim();`